client/.env

```diff
@@ -0,0 +1,2 @@
+EXTEND_ESLINT=true
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
Why are we skipping this check? I understand why we have set `EXTEND_ESLINT` but why the preflight check? If it is to overwrite the eslint config due to our stricter eslint checks now. Have you had a look at this: https://github.com/facebook/create-react-app/issues/5247#issuecomment-439360498

Reply:
I added this comment
```
# Pre-flight check fails as it incorrectly detects the node_modules folder in the root.
# Skip it until we can move the server-specific project files to the server/ folder.
```
https://www.notion.so/Move-backend-package-json-and-other-project-files-to-server-35564724d07645bfbffe461a293c548d